.github/workflows/code-analysis.yml

```diff
@@ -1,10 +1,6 @@
 name: Code Analysis
 
-on:
-  push:
-    branches: [ master, android-v2 ]
-  pull_request:
-    types: [ opened, synchronize, reopened ]
+on: [push, pull_request]
```

Comment:
`on push` will trigger this workflow for each commit, where `code-analysis.yml` is present.
If we do want this, we should remove `on pull_request`, since it triggers this workflow second time for each commit in pull requests.

Reply:
Thanks for the catch. Updated it.